pages/roadmap/[...slug].tsx

```diff
@@ -222,10 +222,10 @@ export default function RoadmapPage(props: InferGetServerSidePropsType<typeof ge
   return (
     <>
       <PageHeader />
-      <div style={{ overflowY: 'auto', height: 'calc(100vh - 100px)' }}>
+      <div style={{ overflowY: 'auto', height: 'calc(100vh - 120px)', paddingTop: '28px' }}>
         {issueDataState.ornull != null && <StarmapsBreadcrumb currentTitle={issueDataState.ornull.title.value} />}
         <ErrorNotificationDisplay errorsState={starMapsErrorsState} issueDataState={issueDataState}/>
-        <Box pt={5} pr={{ base:"30px", sm:"30px", md:"60px", lg:"120px" }} pl={{ base:"30px", sm:"30px", md:"60px", lg:"120px" }} >
+        <Box pr={{ base:"30px", sm:"30px", md:"60px", lg:"120px" }} pl={{ base:"30px", sm:"30px", md:"60px", lg:"120px" }} >
```

Comment:
nit: why is this needed? I'm concerned that the height between this and the `.scrollable` component need to be maintained across separate files, and I would like to see us get to a point where we centralize these calculations and handle this better.

Reply:
this is a valid concern, but note the change did not introduce the height calc. it only updated it to adjust for the margin/padding changes